android/fakewallet/src/main/res/values/constants.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="no_connection_timeout_millis">3000</integer>
```

Comment:
Whether or not integer constants should be defined in XML resources is an open question (and I'm happy to revisit), but IMO we should do it for `fakewallet` as a whole. Right now, there is already a companion object in `MobileWalletAdapterViewModel` with constants defined; let's keep it all centralized there.